### ISSUE TITLE
License file for project

### ISSUE THREAD

**Issue by kelkarn** [scassandra/scassandra-server#144]
Hi Christopher,

It would be nice if you added a license file to the Scassandra project. Or does it already exist somewhere?

**Comment by kelkarn** [comment]
To clarify, I meant a LICENSE.txt for the main project (I just saw that you have license files for the sub-projects for the java client and server projects).

**Comment by kelkarn** [comment]
It would also be nice to see a mention of the Apache v2.0 license nature of the project on the main project website (at the bottom or somewhere) here: http://www.scassandra.org

**Comment by chbatey** [comment]
Good point. Happy to accept a PR for both (site is on github too). Otherwise i'll get to it eventually.
